echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/microsoftteams/MicrosoftTeamsService.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Cerner Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.microsoftteams;
+
+import groovy.transform.Canonical;
+import lombok.extern.slf4j.Slf4j;
+import retrofit.client.Response;
+
+@Canonical
```

Comment:
This is a Groovy annotation. Remove and replace with a constructor.

Reply:
Removed this annotation